keras/optimizers/schedules/learning_rate_schedule.py

```diff
@@ -153,6 +153,9 @@ def __init__(
         self.staircase = staircase
         self.name = name
 
+        if self.decay_steps <= 0:
+            raise ValueError("Argument `decay_steps` must be > 0.")
```

Comment:
print out decay steps.
```
if self.decay_steps <= 0:
  raise ValueError(f"Argument `decay_steps` must be > 0.-- got {self.decay_steps}")
```

Reply:
@innat-asj , Thanks for inputs. DOne the changes as suggested.